src/opm/input/eclipse/Schedule/KeywordHandlers.cpp

```diff
@@ -879,13 +879,13 @@ File {} line {}.)", wname, location.keyword, location.filename, location.lineno)
                   if (is_terminal_node) {
                       if (vfp_table > 0) {
                           std::string msg = fmt::format("The group {} is a terminal node of the network and should not have a vfp table assigned to it.", group_name);
-                          throw OpmInputError(msg, handlerContext.keyword.location());
+                          OpmLog::warning(OpmInputError::format(msg, handlerContext.keyword.location()));
                       }
                       node.terminal_pressure(pressure_item.getSIDouble(0));
                       nodes.push_back(node);
-                      // Remove the  branch upstream if it was part of the network
-                      if (network.has_node(downtree_node) && network.has_node(uptree_node))
-                          network.drop_branch(uptree_node, downtree_node);
+                      // Need to add the flow further up the network in case of other fixed-pressure nodes
+                      if (!uptree_node.empty())
+                        network.add_or_replace_branch(Network::Branch(downtree_node, uptree_node, 9999, 0.0));
```

Comment:
If a group node A is a fixed pressure node, it might have a pressure of 100 bar.  If we have a branch connected to a uptree_node with 9999 vfp table, it means this the node A has the same pressure with the uptree_node. 

Typically this uptree_node can obtain a nodal pressure from its own upstream network. the pressure might not be equal to the nodal pressure assigned to the fixed-pressure node. 

I still think removing the branch will be the correct behavoir.

Reply:
The implementation of the function 
```std::vector<std::reference_wrapper<const Node>> ExtNetwork::roots() const;```
also reflects that a fixed pressure node is considered to be a root node and there is should be no upstream branch from this node. 

But I might miss something by purely looking at the code.

Reply:
From the implementation in the extended network, I can see removing the branch is not totally necessary. And the fixed pressure node can obtain the pressure directly without checking the upstream network. Then it becomes a question whether we want to following the keyword description.

Reply:
No, we need to add this. The standard network is a network in the GRUPTREE structure, and considering the relevant test case with group structure defined [here](https://github.com/OPM/opm-tests/blob/8eecdbdf716f765ffd1ba3751deec7e2faef4fed/network/NETWORK-01-MULTIROOT_STD.DATA#L241): with, GRPB defined as a fixed pressure node in addition to FIELD, the flow from GRPB will not be accounted for in the network from GRPA and upwards unless this branch is added.

Reply:
![image](https://github.com/OPM/opm-common/assets/2436155/3cffeb3b-a89f-492b-84ed-83a20cde517f)

Reply:
> the flow from GRPB will not be accounted for in the network from GRPA and upwards unless this branch is added.

Oh, yes, this makes sense. Thanks for the clarification. I did not consider from the flow perspective.